### PR TITLE
Implement `nin` filter for memory db

### PR DESCRIPTION
**Pull request by 0ff**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by raymondfeng** [comment]
@0ff Please sign the CLA - https://cla.strongloop.com/agreements/strongloop/loopback-datasource-juggler

File: lib/connectors/memory.js [hunk 1]
```diff
@@ -480,6 +480,16 @@ function applyFilter(filter) {
         return false;
       }
 
+      if (example.nin) {
+        // if (!value) return false;
```
Comment (superkhau) [inline lib/connectors/memory.js]: @raymondfeng Shouldn't this comment be removed?

**Comment by rmg** [comment]
@slnode ok to test

**Comment by bajtos** [comment]
Closing as abandoned.

**Pull request closed without merging**